assembl/static2/js/app/actions/actionTypes.js

```diff
@@ -0,0 +1,40 @@
+/* redux action types */
```

Comment:
Can we find a name more verbose for this file, like `localeActionTypes` for example?

Reply:
no, this file is meant to contain for all redux action types